Reflect the latest version of README to Japanese version

0xtailf0xtail opened the pull request:
# Relates to:

Japanese version of README is obsolete. It should be updated based on the latest version of README in English.

Also, some parts of the original README are confusing (e.g. Manual setup).

# Risks

Low. There is no code change in this MR.

# Background

## What does this PR do?

Updates the Japanese version of README based on the latest version of README in English.

Also, this PR fixes the confusing parts of the original README.

## What kind of change is this?

Document updates

## Why are we doing this? Any context or related work?

Making the onboarding process for Japanese developers. Many of them rely on Japanese documentations and keeping them fresh can provide better developer experience for them.

# Documentation changes needed?

This PR itself changes the documentation.

# Testing

Did proof reading it with ChatGPT. Also, reviewed it in Preview of VS Code. 

## Where should a reviewer start?

Original README, then Japanese version.

## Detailed testing steps

N/A

# Deploy Notes
## Database changes

N/A

## Deployment instructions

N/A

## Discord username

chuckk.

nicky-ru committed:
feat: add cache to evm plugin

nicky-ru committed:
fix: align transfer action to it's type

nicky-ru committed:
perf: update evm state on transfer

Commit:
bugfix. the port 80 is not listening use 3000

3000 is listening on the docker image

denizekiz committed:
Seperated imageModelProvider and imageVisionModelProvider for ImageDescriptionService

denizekiz committed:
modelVisionProvider Added to IagentRuntime interface type.ts

mcp97 committed:
Merge branch 'develop' into 711--fix-bug-in-facts_ts

nusk0 committed:
Solve Issue 1643, imageDescriptionService not working with other providers than OpenAI

mcp97 committed:
Merge pull request #1648 from metakai1/711--fix-bug-in-facts_ts

fix: Fix bug in plugin-bootstrap/src/evaluators/facts.ts

mcp97 committed:
Merge branch 'develop' into main

mcp97 committed:
Merge branch 'develop' into main

mcp97 committed:
Merge pull request #1634 from xMariem/main

Add README_AR.md

mcp97 committed:
Merge branch 'develop' into main

mcp97 committed:
Merge pull request #1645 from mdominikd/main

Added Hungarian README

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

shakkernerd committed:
Merge branch 'develop' into pr/cmadaanaya/1577

shakkernerd committed:
Merge pull request #1577 from cmadaanaya/evelyn

fix: activate web-search plugin in agents/

mcp97 committed:
Merge branch 'develop' into 1634--fix_ImageDescriptionService

mcp97 committed:
Merge pull request #1667 from nusk0/1634--fix_ImageDescriptionService

fix: 1634  fix image description service

mcp97 committed:
Merge branch 'develop' into VisionModel2

mcp97 committed:
Merge pull request #1664 from denizekiz/VisionModel2

fix: Seperated imageModelProvider and imageVisionModelProvider for ImageDescriptioServices

mcp97 committed:
Merge branch 'develop' into patch-1

mcp97 committed:
Merge pull request #1660 from 0xRider/patch-1

fix: Update Supabase schema.sql

nusk0 committed:
fix issue 1668, ImageDescriptionService can't get image description from twitter post with images

nusk0 committed:
Merge branch 'main' into 1668--fix_TwitterImageLink

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

shakkernerd committed:
Merge branch 'develop' into 1668--fix_TwitterImageLink

Ed-Marcavage committed:
Merge branch 'develop' of https://github.com/elizaOS/eliza into feature/full-plugin-docs

Ed-Marcavage committed:
Feature/full plugin docs (#52)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation (#5)

* feat: add support for agentic plugin documentation

* Feature/full plugin docs (#7)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Test (#53)

* chore: remove .env

* git ignore has .cursorrules

* Revert "lint: fix EmbeddingProvider already defined error (via claude)"

This reverts commit bbbf3255311c7ac582555ff514b9eb9bbdd43ad0.

* fix(client-twitter): tighten engagement criteria and raise quality thresholds

* Update README.md

* Update README_CN - Model Configuration

The original model configuration guide is misleading, it tells readers to configure model provider by change the `XAI_MODEL` parameters.

But this is for the XAI configurations, i update the description completely with the right configuration steps.

* Added Polish Readme file

* Fixed polish's readme typo's

* Rename Readme_PL.md to README_PL.md

* feat: fix chat client to autoscroll

* chore: update changelog

* feat: improve messages

Improve message so criteria is displayed

* fix number prefixed username, add X brand in messaging

* refactor: reorganize imports in video service

* fix: improve clarity and grammar in pull request template

* fix tg markdown issue.

* fix: update jsdoc automation workflow

* fix: update jsdoc automation workflow

* docs: Add JSDoc documentation to constants.ts

* docs: Add JSDoc documentation to createRuntime.ts

* docs: Add JSDoc documentation to testSetup.ts

* docs: Add JSDoc documentation to types.ts

* docs: Add JSDoc documentation to constants.ts

* docs: Add JSDoc documentation to createRuntime.ts

* docs: Add JSDoc documentation to testSetup.ts

* docs: Add JSDoc documentation to types.ts

* feat: Twitter spaces integration

* Adding pnpm-lock.yaml

* add character file script to make system prompt and templates easier to add to existing character files or generate a new one

* feat: add theme toggle functionality with dark and light mode support

- Imported ThemeToggle component in AppSidebar
- Added ThemeToggle to SidebarFooter in AppSidebar
- Initialized theme based on localStorage and system preferences in main.tsx to support dark and light modes

* docs: Add "What Did You Get Done This Week? 7" notes

* Adding back comments and documentation

* add TEE plugin-env support

* fix

* fix

* fix: init 768 dimension in database for gaianet

* chore: parse files through prettier

* Update README.md

* Add files via upload

* feat: add optional TRANSCRIPTION_PROVIDER setting with fallback logic, moving twitter spaces plugins to Eliza repo

* Updates for web search and timeout

* Cleanup on packages/client-twitter/src/plugins/SttTtsPlugin.ts

* Update agents.md

Minor update to correct "should"

* add an client-direct endpoint to get memories by agentid and roomid

* docs: fix Contributing Guide

* feat: add docs for image generation plugin

* fix: fix the image redirection

* update volcengine model

* feat: Add Spanish Translation for Documentation README

* added README files to all plugins

* use tavily sdk

* file updates

* tweak of evm transfer template

* refactor(plugin-conflux): output detailed invalid content

* add experimental telemetry model option

* chore: console -> elizaLogger

* chore: pnpm lock file

* chore: fix conflicts

* chore: add readme

* fix: set publishedDate to optional as stated by the docs.

* chore: move tavily package to the core

* chore: pnpm lock file

* feat: add error handling for TAVILY_API_KEY key

* chore: remove conflicting files

* chore: README files

* chore: Update local-development.md

* fix continue double responses

* fix google api key

* feat: include priority to plugin-tee build

* chore: remove build cache with clean command

* chore: changes to settings

* fix: add @tavily/core to external libs (almost took my life)

* chore: remove logs

* chore: console -> elizaLogger

* add plugins to the key components section of the faq

during a recent discussion on discord it was not clear to some folks the conceptual difference between clients and plugins. this just adds a bit more high level info up top on the FAQ for future creators.

* re-build fx

* Fix double spaced tweets in Post.ts

Currently, despite the prompting stating to use "\\n\\n (double spaces" between statements the tweets always output new statements with just a single newline, making tweets look blocky. 

For example, tweets previous would look like (despite the prompt): 
"Dobby is so excited to start this new year, full of possibilities and chances to make a difference!
Dobby has big plans to help his dear friends and make the wizarding world an even brighter place."

This ensures the double spaces requested in prompting is applied to the tweets with the tweets now looking like: 
"Dobby must remind all friends to be cautious of dark magic lurking in the shadows.

Dobby's heart aches at the thought of anyone falling prey to its deceitful powers."

This allows agents to tweet better-formatted posts that ease visilibty.

* add TranscriptionProvider

* select transcription provider based on character setting

* clean code

* fix evm deps

* fix bad merge

* remove claude changes

* corrected path for image upload

* Add README_AR.md

Add Arabic language for readme.

* chore: update viem dependency version in plugin-evm and plugin-goat

* Update README.md

add Arabic language reference

* Update package.json

* Update package.json

* fix: remove unwanted gitignore

* chore: add enable web search variable

* chore: pnpm lock file

* fix: import web search plugin only on when enabled

* fix: lockfile

* testing

* update

* Added Hungarian README

* update clean.sh to include deleting lock file to really reset the repo

* update integreation test

* run on pull request not pull request target

* update to use pnpm command

* trigger re-run of workflows

* fix

* revert

* actually need to revert

* Fix bug in plugin-bootstrap/src/evaluators/facts.ts

{{user1}} should be {{user2}} :

-------
in factEvaluator, in the examples template,

{{user1}} should be {{user2}} instead

                {
                    user: "{{user1}}",
                    content: { text: "Which city?" },
                },
                {
                    user: "{{user2}}",
                    content: { text: "Oakland" },
                },
                {
                    user: "{{user1}}",
                    content: {
                        text: "Oh, I've never been there, but I know it's in California",
                    },
                }

* standardize viem dependency so project builds now

* missing depdencies

* revert to target due to https://github.com/ai16z/eliza/pull/993/files#r1883351009

* remove submodule

* Solve Issue 1643, imageDescriptionService not working with other providers than OpenAI

* Update schema.sql

There is an extra semicolon that causes a syntax error when trying to upload the schema to supabase.

* Solve Issue 1643, imageDescriptionService not working with other providers than OpenAI

* Seperated imageModelProvider and imageVisionModelProvider for ImageDescriptionService

* modelVisionProvider Added to IagentRuntime interface type.ts

* Feature/full plugin docs (#52)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation (#5)

* feat: add support for agentic plugin documentation

* Feature/full plugin docs (#7)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

---------

Co-authored-by: J. Brandon Johnson <bjohnson@swizzmagik.com>
Co-authored-by: E.FU <e-fu@users.noreply.github.com>
Co-authored-by: odilitime <janesmith@airmail.cc>
Co-authored-by: SK1989sL <91366180+SK1989sL@users.noreply.github.com>
Co-authored-by: JOMOKING <83915728+RedHorse823@users.noreply.github.com>
Co-authored-by: Judasz <yanushevitz@gmail.com>
Co-authored-by: yanushevitz <75587556+yanushevitz@users.noreply.github.com>
Co-authored-by: Monil Patel <monilpat@gmail.com>
Co-authored-by: Matt Gunnin <mgunnin@esports.one>
Co-authored-by: azep-ninja <anthony@quickintel.io>
Co-authored-by: jin <32600939+madjin@users.noreply.github.com>
Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>
Co-authored-by: slkzgm <contact@slkzgm.com>
Co-authored-by: HashWarlock <hashwarlock@protonmail.com>
Co-authored-by: zkfriendly <zkfriendly@proton.me>
Co-authored-by: Phlo <dev@phlo.info>
Co-authored-by: Shakker Nerd <shakkerdroid@gmail.com>
Co-authored-by: Sam <samuv.eth@gmail.com>
Co-authored-by: csh <458761603@qq.com>
Co-authored-by: tomguluson92 <314913739@qq.com>
Co-authored-by: Chanddeep Madaan <c.madaan@karmiclabs.com>
Co-authored-by: Yorke E. Rhodes III <yorke.rhodes@microsoft.com>
Co-authored-by: treppers <90061012+treppers@users.noreply.github.com>
Co-authored-by: twilwa <spywarezzz@proton.me>
Co-authored-by: 0xFloyd <arfloyd7@gmail.com>
Co-authored-by: chandiniv1 <velamalachandini9@gmail.com>
Co-authored-by: Sebastián Salazar Solano <112297389+salazarsebas@users.noreply.github.com>
Co-authored-by: zhourunlai <xiaorun95@gmail.com>
Co-authored-by: CheddarQueso <cheddarqueso3d@gmail.com>
Co-authored-by: zkvm <kevin.zhuang@bit.com>
Co-authored-by: bendanzhentan <455462586@qq.com>
Co-authored-by: Ninja Dev (QI) <142059473+azep-ninja@users.noreply.github.com>
Co-authored-by: Cole Gillespie <745064+cole-gillespie@users.noreply.github.com>
Co-authored-by: Suicidal Goofy <tr.ipsie.boom@gmail.com>
Co-authored-by: Ting Chien Meng <tcm390@nyu.edu>
Co-authored-by: Shaw <shawmakesmagic@gmail.com>
Co-authored-by: ShreyGanatra <ganatrashrey2002@gmail.com>
Co-authored-by: Mariem Mohamed <63235508+xMariem@users.noreply.github.com>
Co-authored-by: Arthera Node <node@arthera.bask.guru>
Co-authored-by: mdominikd <154684992+mdominikd@users.noreply.github.com>
Co-authored-by: metakai1 <km711@protonmail.com>
Co-authored-by: nusk0 <guillaume1938@hotmail.com>
Co-authored-by: 0xRider <134025586+0xRider@users.noreply.github.com>
Co-authored-by: Your Name <denizekiz93@gmail.com>

github-actions[bot] committed:
chore: update pnpm lockfile

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

chandiniv1 committed:
feat: add docs for story plugin

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Feature/full plugin docs (#54)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation (#5)

* feat: add support for agentic plugin documentation

* Feature/full plugin docs (#7)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Merge branch 'develop' of https://github.com/Ed-Marcavage/eliza into develop

github-actions[bot] committed:
chore: update pnpm lockfile

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

mcp97 committed:
Merge pull request #1672 from chandiniv1/chandini/story-plugin

feat: add docs for story plugin

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Feature/full plugin docs (#59)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation (#5)

* feat: add support for agentic plugin documentation

* Feature/full plugin docs (#7)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

github-actions[bot] committed:
chore: update pnpm lockfile

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Feature/full plugin docs (#63)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation (#5)

* feat: add support for agentic plugin documentation

* Feature/full plugin docs (#7)

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

* feat: add support for agentic plugin documentation

github-actions[bot] committed:
chore: update pnpm lockfile

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Merge branch 'develop' into feature/full-plugin-docs

odilitime committed:
add plugin-web-search

odilitime committed:
wire up web-search plugin

odilitime committed:
update lock file for plugin-web-search

odilitime committed:
Merge branch 'develop' of https://github.com/elizaos/eliza into web-search-fix

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
Merge branch 'feature/full-plugin-docs' of https://github.com/Ed-Marcavage/eliza into feature/full-plugin-docs

odilitime committed:
Merge pull request #1676 from odilitime/web-search-fix

fix: add web search to agent

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

Ed-Marcavage committed:
feat: add support for agentic plugin documentation

odilitime committed:
Merge branch 'develop' into tcm-trimTokens

mcp97 committed:
Merge branch 'develop' into 1668--fix_TwitterImageLink

mcp97 committed:
Merge pull request #1671 from nusk0/1668--fix_TwitterImageLink

fix: 1668  fix twitter image link

9547 committed:
chore(scripts): rename ai16z/eliza to elizaOS/eliza

9547 committed:
chore(docs): rename ai16z/eliza to elizaOS/eliza

9547 committed:
chore(docs): rename ai16z/eliza to elizaOS/eliza

9547 committed:
chore(docs): rename ai16z/eliza to elizaOS/eliza

9547 committed:
chore(docs): rename ai16z/eliza to elizaOS/eliza

9547 committed:
chore(docs): rename ai16z/eliza to elizaOS/eliza

9547 committed:
Merge branch 'develop' into project-name

mcp97 committed:
Merge pull request #1679 from 9547/project-name

0xtailf0xtail committed:
Fix some confusing explanations in README

0xtailf0xtail committed:
Reflect the latest version of README to Japanese version

0xtailf0xtail committed:
Merge branch 'develop' into improve-japanese-readme

gnujoow committed:
Update README_KOR.md

Update README with detailed sections

- Added "Features" section with clear use cases in English and Korean.
- Translated "Note for Windows Users" and "Additional Requirements" into Korean.
- Enhanced "Community & Contact" section with actionable phrases.
- Added contributors section with visual representation.
- Included star history chart for project insights.
- Improved formatting and consistency across all sections.

gnujoow committed:
Update README_KOR.md

gnujoow committed:
Update README_KOR.md

gnujoow committed:
Merge branch 'develop' into patch-1

gnujoow committed:
remove link to another translate section

0xaguspunk committed:
Fix readme

0xaguspunk committed:
Merge develop

9547 committed:
fix(core): trimTokens no need to await

affaan-m committed:
Update index.ts

Making sure plugin has a proper default export at bottom to avoid collisions when used in actions module in char file.

SK1989sL committed:
Update README.md spelling 

Just two little points as want to make it look flawless

v1xingyue committed:
plugin-sui support suiprivatekey private key

v1xingyue committed:
add parse cmdline

nicky-ru committed:
fix: remove duplicated code, add cache expiry constant

nicky-ru committed:
test: align tests to new Wallet provider contructor

shakkernerd committed:
Merge branch 'develop' into plugin-sui-patch

shakkernerd committed:
chore: remove conflicted readme files

shakkernerd committed:
chore: readme for story plugin

shakkernerd committed:
chore: npm -> pnpm

shakkernerd committed:
fix: install packages with no frozen lockfile flag

shakkernerd committed:
chore: clean before installing

shakkernerd committed:
Merge pull request #1695 from elizaOS/fix/smokeTests

fix: smoke tests

shakkernerd committed:
Merge branch 'develop' into plugin-sui-patch

denizekiz committed:
supported url format provided

denizekiz committed:
fix

shakkernerd committed:
Merge pull request #1693 from v1xingyue/plugin-sui-patch

feat: Plugin sui support for suiprivatekey0x account

denizekiz committed:
fix

shakkernerd committed:
Merge branch 'develop' into pr/SK1989sL/1690

shakkernerd committed:
Merge pull request #1690 from SK1989sL/patch-2

docs: update README.md spelling

daizhengxue committed:
Update environment.ts

Fix Twitter username validation regex to match platform requirements

0xaguspunk committed:
Merge develop

nicky-ru committed:
feat: improve transfer template

shakkernerd committed:
Merge branch 'develop' into pr/daizhengxue/1698-1

mcp97 committed:
Merge branch 'develop' into improve-japanese-readme

nicky-ru committed:
chore: remove unnecessary export

shakkernerd committed:
chore: fix typo

monilpat commented on README.md:
This shouldn't be deleted

mcp97 reviewed (changes requested):
Please keep it so just README_JA is updated thank you

shakkernerd committed:
Merge pull request #1698 from daizhengxue/main

chore: twitter username validation message

mcp97 committed:
Merge branch 'develop' into trim-token

mcp97 committed:
Merge branch 'develop' into patch-1

mcp97 committed:
Merge branch 'develop' into patch-2

mcp97 committed:
Merge pull request #1683 from gnujoow/patch-1

Update README_KOR.md: Added missing sections and improved Korean translations for clarity

mcp97 committed:
Merge branch 'develop' into trim-token

mcp97 committed:
Merge pull request #1686 from 9547/trim-token

fix(core): trimTokens no need to await

shakkernerd committed:
Merge branch 'develop' into patch-2

mcp97 committed:
Merge branch 'develop' into patch-2

shakkernerd committed:
Merge pull request #1688 from affaan-m/patch-2

chore: update web search plugin export

denizekiz committed:
shouldUseBase64 fix

nicky-ru committed:
fix: log with elizalogger

0xtailf0xtail committed:
Address to the PR feedback

0xtailf0xtail committed:
Merge branch 'develop' into improve-japanese-readme

0xtailf0xtail committed:
Revert the original README and make only necessary changes

0xtailf0xtail commented on README.md:
Sorry! Seems like I messed up the README.md when I modified it. I reverted it back and left only changes I wanted to suggest.
Please let me know if you would like to keep this or not. If you think this change doesn't make sense, I will revert README entirely! 

I appreciate your review!

0xaguspunk committed:
Merge branch 'develop' into goat-v2

mcp97 committed:
Merge pull request #1403 from goat-sdk/goat-v2

feat: improve GOAT integration by allowing tool calling when using generateText

dxlliv committed:
Simulate discord typing while generating a response

aalimsahin committed:
refactor: modularize mutations

aalimsahin committed:
refactor: modularize queries

aalimsahin committed:
Merge branch 'develop' into refactor/client-react-query

tcm390 committed:
Merge branch 'develop' into tcm-trimTokens

odilitime committed:
Merge branch 'develop' into fix/postgres-adapter-schema-check

odilitime committed:
Merge pull request #1377 from ryanleecode/fix/postgres-adapter-schema-check

fix: postgres-adapter - remove nonsensical schema check

odilitime committed:
Merge branch 'develop' into fix/disable-trust-provider-for-postgres

odilitime committed:
Merge branch 'develop' into tcm-trimTokens

odilitime committed:
Merge pull request #1536 from swizzmagik/fix/disable-trust-provider-for-postgres

fix: disable trust provider for PostGres db

odilitime committed:
Merge branch 'develop' into tcm-trimTokens

9547 committed:
chore(zksync-era): rm not used imports

9547 committed:
Merge branch 'develop' into zksync-warning

shakkernerd committed:
chore: pnpm lock file

shakkernerd committed:
chore: remove double imports

shakkernerd committed:
chore: remove redundant web search implementation

shakkernerd committed:
chore: remove redundant env

shakkernerd committed:
Merge pull request #1718 from elizaOS/fix/agent_index

fix: multiple web search import in agent

shakkernerd committed:
Merge branch 'develop' into zksync-warning

shakkernerd committed:
Merge pull request #1716 from 9547/zksync-warning

chore(zksync-era): rm not used imports

marcellodesales committed:
:bug: fix plugins.md formatting for docs

Can't build the docs because of the following:

docs/docs/packages/plugins.md

39.56 [webpackbar] ✔ Client: Compiled with some errors in 37.40s
39.56 [ERROR] Client bundle compiled with errors therefore further build is impossible.
39.56 Error: MDX compilation failed for file "/opt/docusaurus/docs/packages/plugins.md"
39.56 Cause: Unexpected lazy line in expression in container, expected line to be prefixed with `>` when in a block quote, whitespace when in a list, etc
39.56 Details:
39.56 {
39.56   "column": 1,
39.56   "message": "Unexpected lazy line in expression in container, expected line to be prefixed with `>` when in a block quote, whitespace when in a list, etc",
39.56   "line": 617,
39.56   "name": "617:1",
39.56   "place": {
39.56     "_bufferIndex": 0,
39.56     "_index": 10,
39.56     "line": 617,
39.56     "column": 1,
39.56     "offset": 21207
39.56   },
39.56   "reason": "Unexpected lazy line in expression in container, expected line to be prefixed with `>` when in a block quote, whitespace when in a list, etc",
39.56   "ruleId": "unexpected-lazy",
39.56   "source": "micromark-extension-mdx-expression",
39.56   "url": "https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#unexpected-lazy-line-in-expression-in-container-expected-line-to-be-prefixed"
39.56 }
39.56
39.56 --------------------------

odilitime committed:
add echochambers to agent

odilitime committed:
Merge branch 'develop' into fix-echo

marcellodesales committed:
:recycle: add packageManager to docs/package.json

 > [prod 3/4] RUN pnpm ci:
0.645 /usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:22090
0.645       throw new UsageError(`No version specified for ${raw2} in "packageManager" of ${source}`);
0.645             ^
0.645
0.645 UsageError: No version specified for pnpm in "packageManager" of package.json
0.645     at parseSpec (/usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:22090:13)
0.645     at loadSpec (/usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:22164:11)
0.645     at async Engine.findProjectSpec (/usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:22354:22)
0.645     at async Engine.executePackageManagerRequest (/usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:22410:24)
0.645     at async Object.runMain (/usr/local/lib/node_modules/corepack/dist/lib/corepack.cjs:23102:5) {
0.645   clipanion: { type: 'usage' }
0.645 }

marcellodesales committed:
:bug: fix missing packages from docs/package.json

While trying to build within a docker image, here are the missing packages.

40.12
40.12 --------------------------
40.12
40.12 Module not found: Error: Can't resolve 'lunr' in '/opt/docusaurus/.docusaurus'
40.12
40.12 --------------------------
40.12
40.12 Module not found: Error: Can't resolve '@docusaurus/theme-common' in '/opt/docusaurus/community/components'
40.18  ELIFECYCLE  Command failed with exit code 1.
------
failed to solve: process "/bin/sh -c pnpm run build" did not complete successfully: exit code: 1

shakkernerd committed:
Merge pull request #1719 from odilitime/fix-echo

fix: add echochambers to agent

shakkernerd committed:
chore: remove unused imports

tcm390 committed:
Default to TikToken truncation using the gpt-4o-mini model if tokenizer settings are not defined

marcellodesales committed:
:whale: :new: add Dockerfile.docs and update docker-compose

Since the docs dependend on the files from packages, we have
to include the Dockerfile.docs in the root directory because
of the build context.

Adding a new docker-compose-docs.yaml to always build the docs from
the file directly. Just adding a new service for the docs:

$ docker compose up --build
[+] Building 55.5s (20/20) FINISHED                                                                                                                      docker:desktop-linux
 => [docs internal] load build definition from Dockerfile.docs                                                                                                           0.0s
 => => transferring dockerfile: 1.88kB                                                                                                                                   0.0s
 => [docs] resolve image config for docker.io/docker/dockerfile:1                                                                                                        1.7s
 => CACHED [docs] docker-image://docker.io/docker/dockerfile:1@sha256:93bfd3b68c109427185cd78b4779fc82b484b0b7618e36d0f104d4d801e66d25                                   0.0s
 => [docs internal] load build definition from Dockerfile.docs                                                                                                           0.0s
 => [docs internal] load metadata for docker.io/library/node:23.3.0-slim                                                                                                 1.3s
 => [docs internal] load .dockerignore                                                                                                                                   0.0s
 => => transferring context: 2B                                                                                                                                          0.0s
 => [docs base 1/4] FROM docker.io/library/node:23.3.0-slim@sha256:8b30809f66a6ea8896b9a5d004b4fe2cc0e8061d981d3784fb0e80a19b86ab9d                                      0.0s
 => [docs internal] load build context                                                                                                                                   0.1s
 => => transferring context: 409.46kB                                                                                                                                    0.1s
 => CACHED [docs base 2/4] RUN corepack enable                                                                                                                           0.0s
 => CACHED [docs base 3/4] WORKDIR /opt/docusaurus                                                                                                                       0.0s
 => CACHED [docs base 4/4] RUN apt-get update && apt-get install -y git                                                                                                  0.0s
 => CACHED [docs prod 1/8] WORKDIR /opt/docusaurus                                                                                                                       0.0s
 => CACHED [docs prod 2/8] COPY docs/package.json /opt/docusaurus/package.json                                                                                           0.0s
 => CACHED [docs prod 3/8] COPY docs/package-lock.json /opt/docusaurus/package-lock.json                                                                                 0.0s
 => CACHED [docs prod 4/8] RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install                                                                                0.0s
 => CACHED [docs prod 5/8] COPY docs/ /opt/docusaurus/                                                                                                                   0.0s
 => CACHED [docs prod 6/8] COPY packages/ /opt/packages/                                                                                                                 0.0s
 => [docs prod 7/8] COPY .git/ /opt/.git/                                                                                                                                0.1s
 => [docs prod 8/8] RUN pnpm run build                                                                                                                                  51.4s
 => [docs] exporting to image                                                                                                                                            0.8s
 => => exporting layers                                                                                                                                                  0.8s
 => => writing image sha256:0a580f2889191121880df85b8951563386f838f7e5f8723dec750ce292334498                                                                             0.0s
 => => naming to docker.io/library/eliza-docs                                                                                                                            0.0s
[+] Running 1/0
 ✔ Container eliza-docs-1  Recreated                                                                                                                                     0.0s
Attaching to docs-1
docs-1  | (node:1) ExperimentalWarning: CommonJS module /usr/local/lib/node_modules/npm/node_modules/debug/src/node.js is loading ES Module /usr/local/lib/node_modules/npm/node_modules/supports-color/index.js using require().
docs-1  | Support for loading ES Module in require() is an experimental feature and might change at any time
docs-1  | (Use `node --trace-warnings ...` to show where the warning was created)
docs-1  |
docs-1  | > eliza-docs@0.1.7-alpha.2 serve
docs-1  | > docusaurus serve --host 0.0.0.0 --no-open
docs-1  |
docs-1  | [SUCCESS] Serving "build" directory at: http://0.0.0.0:3000/eliza/

tcm390 committed:
Remove 'model' parameter from trimTokens function; allow tokenizer model to be configured via environment settings

tcm390 committed:
Remove model parameter

marcellodesales committed:
:books: add section to docs/README.md for dockerized docs

madjin committed:
Merge branch 'develop' into feature/full-plugin-docs

sin-bufan committed:
Make the unit of ACTION_INTERVAL consistent across all instances to minutes.

osrm committed:
Update README_KOR.md

Fix language and grammar to match native Korean

Commit:
feat: support for eternalai provider can write request/response

Commit:
feat: support for eternalai provider can write request/response

Commit:
feat: support for eternalai provider can write request/response

sin-bufan committed:
Merge branch 'develop' into fix-twitter-actionInterval

AIFlowML committed:
fix(client-slack): implement Media type properties in message attachments - #1384

Commit:
Merge branch 'develop' into eternal-ai-pr

AIFlowML committed:
fix(postgres): Handle vector extension creation properly (#1561)

- Add proper checks for vector extension existence

- Create extensions schema if not exists

- Add extensions schema to search path

- Ensure idempotent schema creation

sinecose committed:
feat(near): replace console.log to elizaLogger

sinecose committed:
feat(near/transfer): replace console.log to elizaLogger

sinecose committed:
feat(near/wallet): replace console.log to elizaLogger

TresFlames committed:
Update faq.md

Removed mentions of some items.

tcm390 committed:
use 4o as default model

tcm390 committed:
Merge branch 'develop' into tcm-trimTokens

tcm390 committed:
use elizaLogger

shakkernerd committed:
Merge branch 'develop' into Docs-Patch-1

shakkernerd committed:
Merge pull request #1746 from elizaOS/Docs-Patch-1

docs: update faq.md

marcellodesales committed:
Merge branch 'develop' into feature/dockerized-docs-support

shakkernerd committed:
Merge branch 'develop' into kor.md-modify

shakkernerd committed:
Merge pull request #1739 from osrm/kor.md-modify

docs: update README_KOR.md

marsic3 committed:
feat: add Serbian README translation and enhance main README with new features and setup instructions

tcm390 committed:
move trimTokens to core

shakkernerd committed:
Merge branch 'develop' into marsic3/add-serbian

shakkernerd committed:
Merge pull request #1757 from marsic3/marsic3/add-serbian

docs: add Serbian README translation

shakkernerd committed:
Merge branch 'develop' into fix/lint_errors

ryanleecode committed:
Merge remote-tracking branch 'origin/develop' into feat/live-monorepo-types

tcm390 committed:
clean code

marcellodesales committed:
Merge branch 'develop' into feature/dockerized-docs-support

tcm390 committed:
restore test

tcm390 committed:
clean code

tcm390 committed:
remove tokenizer service

gmh5225 committed:
fix(client): improve Windows compatibility for Vite dev server

- Add host configuration to allow IP access
- Use explicit IP address instead of localhost
- Support dynamic server port from environment variables

tcm390 committed:
fall back if unsupported type

tcm390 committed:
Merge branch 'develop' into tcm-trimTokens

odilitime committed:
Merge branch 'develop' into fix/slack-1384-media-type-attachments

tcm390 committed:
Move encoding into try block to handle potential errors during model encoding.

gmh5225 committed:
Merge branch 'develop' into fix-windows

odilitime committed:
Merge branch 'develop' into fix/postgres-1561-vector-extension

shakkernerd committed:
feat: add JsDoc to trimTokens function

shakkernerd committed:
feat: add validation to trimTokens

shakkernerd committed:
Merge pull request #1605 from elizaOS/tcm-trimTokens

fix: Support for Non-OpenAI Models in Token Trimming

ryanleecode committed:
Merge branch 'develop' into feat/live-monorepo-types

shakkernerd committed:
Merge branch 'develop' into fix/lint_errors

shakkernerd committed:
Merge pull request #1759 from elizaOS/fix/lint_errors

fix: build lint errors

shakkernerd committed:
Merge branch 'develop' into fix-windows

shakkernerd committed:
Merge pull request #1760 from gmh5225/fix-windows

fix(client): improve Windows compatibility for Vite dev server

shakkernerd committed:
Merge branch 'develop' into feat/live-monorepo-types

shakkernerd committed:
Merge pull request #1365 from ryanleecode/feat/live-monorepo-types

feat: use custom conditions for live monorepo types

shakkernerd committed:
Merge remote-tracking branch 'origin/develop' into pr/genesis-0000/1740

shakkernerd committed:
fix: handle env using runtime getSetting function

mcp97 committed:
Merge branch 'develop' into console-to-elizaLogger

mcp97 committed:
Merge pull request #1745 from sinecose/console-to-elizaLogger

feat(plugin-near): replace console.log to eliza logger

dxlliv committed:
Merge branch 'develop' into main

madjin committed:
Merge branch 'develop' into feature/full-plugin-docs

madjin committed:
Merge pull request #1675 from Ed-Marcavage/feature/full-plugin-docs

Agentic Eliza Plugin Documenter - Multilingual (e.g., English, Spanish, French)

swizzmagik committed:
fix: line break handling in chat

madjin committed:
Merge branch 'develop' into feature/dockerized-docs-support

madjin committed:
Merge pull request #1722 from marcellodesales/feature/dockerized-docs-support

🐛 fix plugins.md formatting for docs with dockerized docs validation

shakkernerd committed:
Merge branch 'develop' into fix/line-break-formatting-in-chat

shakkernerd committed:
Merge pull request #1784 from swizzmagik/fix/line-break-formatting-in-chat

fix: line break handling in chat

Rudrakc committed:
Removed FerePro plugin

shakkernerd committed:
chore: remove unused import and var

shakkernerd committed:
Merge branch 'develop' into fix/lint_errors

shakkernerd committed:
Merge pull request #1797 from elizaOS/fix/lint_errors

chore: remove unused import and var

mcp97 committed:
Merge branch 'develop' into fix-twitter-actionInterval

mcp97 committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge branch 'develop' into pr-1795

shakkernerd committed:
Merge branch 'develop' into fix/slack-1384-media-type-attachments

shakkernerd committed:
Merge pull request #1741 from AIFlowML/fix/slack-1384-media-type-attachments

fix(client-slack): implement Media type properties in message attachments #1384

mcp97 committed:
Merge branch 'develop' into fix-twitter-actionInterval

shakkernerd committed:
Merge branch 'develop' into fix/postgres-1561-vector-extension

shakkernerd committed:
Merge pull request #1743 from AIFlowML/fix/postgres-1561-vector-extension

fix(postgres): Handle vector extension creation properly (#1561)

shakkernerd committed:
Revert "fix(postgres): Handle vector extension creation properly (#1561)"

shakkernerd committed:
Merge pull request #1799 from elizaOS/revert-1743-fix/postgres-1561-vector-extension

Revert "fix(postgres): Handle vector extension creation properly (#1561)"

shakkernerd committed:
Merge branch 'develop' into fix-twitter-actionInterval

shakkernerd committed:
Merge pull request #1738 from sin-bufan/fix-twitter-actionInterval

fix: standardize ACTION_INTERVAL unit to minutes in twitter client

shakkernerd committed:
Merge branch 'develop' into refactor/client-react-query

shakkernerd committed:
Merge pull request #1713 from aalimsahin/refactor/client-react-query

refactor: client api

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #1712 from dxlliv/main

feat: Simulate discord typing while generating a response

shakkernerd committed:
Merge branch 'develop' into pr-1616

shakkernerd committed:
chore: install with no frozen-lockfile flag

shakkernerd committed:
Merge pull request #1802 from elizaOS/shakkernerd-patch-1

chore: install with no frozen-lockfile flag

shakkernerd committed:
fix: generatioon tests for trimTokens

shakkernerd committed:
Merge pull request #1803 from elizaOS/fix/tests

fix: generation tests for trimTokens

shakkernerd committed:
chore: bump version to v.0.1.7

shakkernerd committed:
Merge pull request #1804 from elizaOS/new_version_0.1.7

chore: bump version to v.0.1.7

0xtailf0xtail committed:
Merge branch 'develop' into improve-japanese-readme

 Conflicts:
	README.md

0xtailf0xtail committed:
Revert the original README and apply the latest changes to Japanese version

0xtailf0xtail commented on README.md:
Hi @monilpat , I changed my mind. The improvements of English README should have been done in the separated PR. Sorry for the confusion. 

I reverted the README as you suggested and just leave Japanese translation in this PR. I hope this makes sense to you!

Thanks again for your review!
